Client/src/Pages/Incidents/index.css

```diff
@@ -0,0 +1,33 @@
+.maintenance-checklist-main {
+  max-width: 350px;
+  min-height: 390px;
+  margin: auto;
+  padding: 5px;
+}
+
+.maintenance-image {
+  width: 280px;
+  height: 150px;
+  background-color: whitesmoke;
+  border-radius: 10px;
+}
+
+.maintenance-title {
+  font-size: 20px;
+  font-weight: bold;
+  margin-top: 40px;
+  margin-bottom: 30px;
+}
```

Comment:
The font color of the text should be `#475467` and a size of `16px`. Please see the Figma design.

In terms of measurements, stuff like paddings, margins, font-sizes and what not, we aim to stick closely to the original Figma design. Makes life easier when you're not always guesstimating. We have some pre-defined variables in `:root` inside the `Client/src/index.css` file, or if you are more inclined to use inline styling we also have a theme schema set up, in `Client/src/Utils/Theme.js`. I would recommend looking over those and applying the styles where possible.

Reply:
Fixed.